vic778/deel_clone
# Rspec model

- Migrate contract
- Add contract to admin pannel
- Create enum for name
- Create name and add user as reference
- Add contract route
- Merge the contract to current user
- Fctory for contract
- Add contract_details_id
- Migrate contractDetails
- Add contract_details to admin pannel
- Validats presence of value and associate to conytract
- Init Factory
- Gems for date
- Gems for date
- Add contract_detail_id
- Add contract_detail_id
- Add resouces
- Associate to contract_details
- Validate presence of date
- Logs user before any action
- Generate date_validation gem
- Has many companies
- Create a route for comanies
- Add companie to admin pannel
- Find current_user trought the profile
- Validate and make unique some values
- Migrate company model
- Fix linters error
- Add contract_id
- Add compani id and user id
- Associate contract and orfile
- Associate to company
- Associate to company trought contract
- Migrate association
- Add comanie to contract
- Fix linters errors
- Fix linters error
- Add payement is
- Associate payement
- Migrate payement
- Associate to user and contract
- Associate to user and contract
- FIx linters
- Fix linters
- Add bygem
- Add bygem
- Fin user and contract before ction
- Add payemtn route
- Create dashboard controller
- Check if the use ris in the contract
- Init help
- Init help
- Init help
- Init help
- Init help
- Init help
- Add status
- FInd payemtn by id
- Create history action
- Create status
- Ad dashboard route
- Migrate status model to payement
- Defined the status by default
- Fix linter error
- Fix linter error
- Fix linter error
- Fix linters error
